bson/src/main/org/bson/codecs/configuration/CodecRegistries.java

```diff
@@ -30,6 +32,22 @@
  */
 public final class CodecRegistries {
 
+    /**
+     * Apply given {@link UuidRepresentation} to the given {@link CodecRegistry}.
+     *
+     * @param codecRegistry the code registry
+     * @param uuidRepresentation the uuid representation
+     * @since 4.5
+     */
+    public static CodecRegistry withUuidRepresentation(final CodecRegistry codecRegistry, final UuidRepresentation uuidRepresentation) {
+        if (codecRegistry instanceof OverridableUuidRepresentationCodecRegistry
```

Comment:
Note that due to it being public the contract has changed a bit.  In the internal API, we could assume that the registry passed to `CodecRegistryHelper#createRegistry` was not itself an instance of  `OverridableUuidRepresentationCodecRegistry`.  But with the new method being public, we can't.  So the check becomes this instead.  So effectively, calling:

   withUuidRepresentation(withUuidRepresentation(registry, STANDARD), STANDARD)

is a no-op.